modules/integration_azure-container-instance/conf/00-hearbeat.yaml

```diff
@@ -0,0 +1,11 @@
+module: Azure Container Instance
+name: heartbeat
+filtering: "filter('resource_type', 'Microsoft.ContainerInstance/containerGroups') and filter('primary_aggregation_type', 'true')"
+transformation: false
+aggregation: true
```

Comment:
This should be a mean by name, rg, region

Reply:
I don't think this what he wants.

In my opinion this seems far safer to group by something "above" a container because in my mind a container is a short lived / dynamic resource where disappearing should not be a cause of legitimate alert.

but if I right understood he wants to have an alert as soon as container disappears. In this case, using group by breaks this BUT it will require a note about this (see https://github.com/claranet/terraform-signalfx-detectors/pull/226#discussion_r552024954)

Reply:
Aggregation function added